### PR TITLE
feat: support number and string for dimensions

### PR DESCRIPTION
**Pull request by Michael-Liendo**
for the capality for use all types of size on css

### CODE REVIEW

**Comment by Michael-Liendo** [comment]
#42

**Comment by EstebanBorai** [comment]
@Michael-Liendo hi! Thanks for openning this issue! Can you share a bit of detail on this please? Whats the problem we are trying to solve here?

**Comment by Michael-Liendo** [comment]
I think that being a text type, you can use all the different types of spacing in css, such as pixels, percentages and other units of measurement in css to obtain the desired handling of the placeholder component

File: src/lib/components/Placeholder.svelte [hunk 1]
```diff
@@ -1,107 +1,107 @@
...
+  export let height = '200';
 
-	/**
-	 * SVG Element `width` attribute value
-	 */
-	export let width = 200;
+  /**
+   * SVG Element `width` attribute value
+   */
+  export let width = '200';
```
Comment (EstebanBorai) [inline src/lib/components/Placeholder.svelte:44]: To keep backwards compat please use `number | string` as type for `height` and `width`.

**Review by EstebanBorai — APPROVED** [review]
lgtm!

**Pull request merged**